Course_Part_10/src/TicTacToeBasic.java

```diff
@@ -0,0 +1,247 @@
+import java.sql.Array;
+import java.util.*;
+
+
+public class TicTacToeBasic {
+
+    static TreeMap<String, String> playOptions;
+    static Scanner input = new Scanner(System.in);
+    static String playMode;
+
+    public static void main(String[] args) {
+
+        playOptions = new TreeMap<>() {{
+            put("A1", "  ");
+            put("A2", "  ");
+            put("A3", "  ");
+            put("B1", "  ");
+            put("B2", "  ");
+            put("B3", "  ");
+            put("C1", "  ");
+            put("C2", "  ");
+            put("C3", "  ");
+        }};
+
+        System.out.println("Welcome to Tic-Tac-Toe");
+        displayBoardStartGame();
+        ChoosePlayMode();
+        displayBoard();
+
+        for (int turn = 1; turn < 10; ) {
+
+            playerTurn();
+            displayBoard();
+            if (isWin()) break;
+            turn++;
+
+            if (isDraw(turn)) break;
+
+            computerTurn();
+            displayBoard();
+            if (isWin()) break;
+            turn++;
+
+        }
+    }
+
+    /**
+     * Let player choose one of two play modes
+     * A: will show all the made moves and all the remaining options during the game
+     * B: will only show all the made moves during the game
+     * The selected value will be stored in the variable PlayMode
+     */
+    static void ChoosePlayMode() {
+        System.out.println();
+        System.out.println("Game has two play modes\n" +
+                "A:                 B:\n" +
+                " A1 | XX | C1          | XX |    \n" +
+                "--------------     --------------\n" +
+                " A2 | B2 | XX          |    | XX \n" +
+                "--------------     --------------\n" +
+                " OO | B3 | C3       OO |    |      "
+        );
+        do {
+            System.out.print("Please choose playmode A or B:");
+            playMode = input.next();
+        } while (!playMode.equals("A") && !playMode.equals("B"));
```

Comment:
dit kun je ook anders schrijven namelijk: 

```
!(playMode.equals("A") || playMode.equals("B"));
```

of te wel: niet A of B, ipv niet A en niet B

de wet die ik hier toepas is de Wet van Morgan vernoemd naar de Britse wiskundige Augustus De Morgan. 

=> https://nl.wikipedia.org/wiki/Wetten_van_De_Morgan

Reply:
interesting, thanks